docs/CodingStyleGuidelines.md

```diff
@@ -98,10 +98,10 @@ Files should contain the following, in order:
 | Location             | Item Type           | Item Name           | File Name              |
 | -------------------- | ------------------- | ------------------- | ---------------------- |
 | `verif/vkits/<vkit>` | Package             | `<vkit>_pkg`        | `<vkit>_pkg.sv`        |
-| `verif/vkits/<vkit>` | Interface           | `<iface_name>_intf` | `<iface_name>_intf.sv` |
+| `verif/vkits/<vkit>` | Interface           | `<iface_name>_if`   | `<iface_name>_if.sv`   |
 | `verif/vkits/<vkit>` | Class               | `<classname>_c`     | `<vkit>_<classname>_c` |
 | `verif/vkits/<vkit>` | Flist File          | N/A                 | `<sim>.flist`          |
-| `verif/<tb>`         | Top-level Testbench | `<tb>_tb_top`       | `<tb>_tb_top.sv`       |
+| `verif/<tb>`         | Top-level Testbench | `<tb>_tb`           | `<tb>_tb.sv`           |
```

Comment:
I believe we have used tb and not tb_top, so I changed this as well. Another alternative is to change the module name. Or maybe I have misunderstood what we are referring to here and there should be no change.

Reply:
Good catch